tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -414,19 +425,25 @@ func TestNewMetrics(t *testing.T) {
 }
 
 func TestDenials(t *testing.T) {
+	testDenials(t, denialRule)
```

Comment:
Please add 2 top level tests `func TestIngressDenial`  and `func TestDenial`.
This way there is better visibility about what exactly failed.

Reply:
Done.